client/hotrod-client/src/test/java/org/infinispan/client/hotrod/configuration/ConfigurationTest.java

```diff
@@ -550,6 +561,17 @@ private void validateConfiguration(Configuration configuration) {
       assertEqualsConfig(NearCacheMode.INVALIDATED, NEAR_CACHE_MODE, configuration);
       assertEqualsConfig(10_000, NEAR_CACHE_MAX_ENTRIES, configuration);
       assertEqualsConfig("near.*", NEAR_CACHE_NAME_PATTERN, configuration);
+      assertEquals(2, configuration.clusters().size());
+      assertEquals(configuration.clusters().get(0).getClusterName(), "siteA");
```

Comment:
Easier to understand! But assertions order is not correct. "expected" is first and "actual" after.

Eventually, if you want to avoid duplication maybe you can refactor like that but as you wish!
```java 
siteA = configuration.clusters().get(0)
assertEquals("siteA",siteA.getClusterName());
siteB = configuration.clusters().get(1)
assertEquals("siteB",siteB.getClusterName());
```
``` java
assertSiteClusterConfiguration(siteA, "hostA1", 11222, 0);
assertSiteClusterConfiguration(siteA, "hostA2", 11223, 1);
assertSiteClusterConfiguration(siteB, "hostB1", 11222, 0);
assertSiteClusterConfiguration(siteB, "hostB2", 11223, 1);

private void assertSiteClusterConfiguration(site, String expectedHostName, int expectedPort, int index) {
      assertEquals(expectedHostName, site.getCluster().get(index).host());
      assertEquals(expectedPort, site.getCluster().get(index).port());
}
```